toktok/integrations.md

```diff
@@ -28,25 +34,43 @@ yourself to this page.
 *   [Antidote](https://github.com/Antidote-for-Tox/Antidote) (iOS)
 *   [Antox](https://github.com/Antox/Antox) (Android)
 *   [Cyanide](https://github.com/krobelus/cyanide) (SailfishOS)
+*   [Harbour-tox](https://github.com/emmanuelduv/harbour-tox) (Sailfish on Jolla)
 *   [ToxDroid](https://github.com/suzp1984/ToxDroid) (Android)
 *   [TRIfA](https://github.com/zoff99/ToxAndroidRefImpl) (Android)
 
 # Bots
 
+*   [CSbot](https://github.com/JFreegman/CSbot)
+*   [Echobot 1](https://github.com/alexbakker/EchoBot)
+*   [Echobot 2](https://wiki.tox.chat/developers/client_examples/echo_bot)
+*   [Echobot 3](https://github.com/toxon/ToxEcho)
+*   [Echobot 4](https://gist.github.com/nurupo/8c6c1d455b39b0830668)
+*   [Echobot 5](https://github.com/yodakohl/c-toxecho)
 *   [Java echo bot](https://github.com/TokTok/echobot-jvm)
+*   [Java echo bot in pure Java](https://github.com/zoff99/java_toxclient_example)
+*   [Skupina-robot](https://github.com/zoff99/skupina-robot)
+*   [ToxBildGruppenArbeiter](https://github.com/zoff99/ToxBildGruppenArbeiter)
+    (Video Conferencing bot for multiple Users)
+*   [ToxBot](https://github.com/JFreegman/ToxBot)
 *   [Video streaming bot](https://github.com/TokTok/streambot-jvm)
 
 # Language bindings
 
+*   [Bash](https://github.com/mensinda/bash-toxcore)
 *   [C#](https://github.com/alexbakker/SharpTox)
 *   [Go](https://github.com/codedust/go-tox) go-tox
 *   [Go](https://github.com/josephyzhou/libtox) libtox
 *   [Go](https://github.com/kitech/go-toxcore) go-toxcore
 *   [Go](https://github.com/xhebox/gtox) gtox
 *   [Haskell](https://github.com/TokTok/hs-toxcore-c)
+*   [Java](https://github.com/zoff99/java_toxclient_example/tree/master/jni-c-toxcore)
 *   [Java/Scala](https://github.com/TokTok/jvm-toxcore-c)
 *   [JavaScript](https://github.com/TokTok/js-toxcore-c)
+*   [Julia](https://github.com/JuliaIO/Toxcore.jl)
+    (Attempt to make Toxcore accessible in Julia)
+*   [Node](https://github.com/saneki-discontinued/node-toxcore)
```

Comment:
This is js-toxcore-c in toktok, but an old discontinued and broken version. "Node == JS".

Reply:
removed